lib/feedjira/parser/podcast.rb

```diff
@@ -19,6 +19,7 @@ class Podcast
       element :description
       element :language
       element :copyright
+      element :author
```

Comment:
`author` is not at channel level, or at least it shouldn't be.
http://cyber.harvard.edu/rss/rss.html